client/pages/sections/granted/summary-table.js

```diff
@@ -1,91 +1,168 @@
+        <tr onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave} onClick={onClick} className={classnames({ expandable: ExpandingRow, hover, expanded })}>
+          {
+            (speciesIndex === 0) && (
+              <Fragment>
+                <td rowSpan={species.length || 1} className="is-last-row">
+                  <span className={classnames({ expanding: ExpandingRow, expanded })}>{ index + 1 }</span>
+                </td>
+                <td rowSpan={species.length || 1} className="is-last-row">{ protocol.title }</td>
+              </Fragment>
+            )
+          }
+          <td className={classnames({ 'is-last-row': isLastRow })}>{ s.name }</td>
           {
             isLegacy
-              ? <th>Est. numbers</th>
+              ? <td>{s.quantity}</td>
               : (
                 <Fragment>
-                  <th>Max. no. of animals</th>
-                  <th>Max. no. of uses per animal</th>
+                  <td className={classnames({ 'is-last-row': isLastRow })}>{ s['maximum-animals'] }</td>
+                  <td className={classnames({ 'is-last-row': isLastRow })}>{ s['maximum-times-used'] }</td>
                 </Fragment>
               )
           }
-          <th>Life stages</th>
-          <th>GA?</th>
-          <th>Severity category</th>
-        </tr>
-      </thead>
-      <tbody>
-        {
-          (protocols || []).map((protocol, index) => {
-            const species = !isLegacy
-              ? filterSpeciesByActive(protocol, project)
-              : (protocol.species || []).map(s => {
-                const label = getLegacySpeciesLabel(s);
-                return {
-                  ...s,
-                  name: label || '-'
-                };
-              });
-            return species.map((s, speciesIndex) => (
-              <tr key={index + speciesIndex}>
-                {
-                  (speciesIndex === 0) && (
-                    <Fragment>
-                      <td rowSpan={species.length || 1}>{ index + 1 }</td>
-                      <td rowSpan={species.length || 1}>{ protocol.title }</td>
-                    </Fragment>
-                  )
-                }
-                <td>{ s.name }</td>
-                {
-                  isLegacy
-                    ? <td>{s.quantity}</td>
-                    : (
-                      <Fragment>
-                        <td>{ s['maximum-animals'] }</td>
-                        <td>{ s['maximum-times-used'] }</td>
-                      </Fragment>
-                    )
-                }
-                <td>
-                  {
-                    isLegacy
-                      ? s['life-stages']
-                      : (s['life-stages'] || []).join(', ')
-                    }
-                </td>
-                {
-                  isLegacy && <td>{s['genetically-altered'] === true ? 'Yes' : 'No'}</td>
-                }
+          <td className={classnames({ 'is-last-row': isLastRow })}>
```

Comment:
As mentioned above, using `:last-child` for this would be preferable.

Reply:
Oh wait, this isn't _actually_ the last row is it? It's the last _species_. Is that right?

Reply:
yes - that - technically it is a row, but there can b n rows per species